ImagePicker/ImagePickerController+Delegate.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  ImagePickerController+Delegate.swift
+//  ImagePicker
+//
+//  Created by Anna Shirokova on 13/06/2018.
+//  Copyright © 2018 Inloop. All rights reserved.
+//
+
+import Foundation
+import  Photos
+
+extension ImagePickerController : ImagePickerDelegateDelegate {
+    func imagePicker(delegate: ImagePickerDelegate, didSelectActionItemAt index: Int) {
+        self.delegate?.imagePicker(controller: self, didSelectActionItemAt: index)
```

Comment:
Two different delegates, probably it will be better to rename one of them. But they are both public.. What about this?
```swift
func imagePicker(delegate pickerDelegate: ImagePickerDelegate, didSelectActionItemAt index: Int) {
   ...
}